test/JDBC/README.md

```diff
@@ -1,5 +1,377 @@
+GO
+```
+
+Input file type: `.mix`
+
+---
+
+### **IMPORTANT**
+- If you want to execute a SQL Batch in `.txt` input files, you will need to specify the batch in a single line without the `GO` batch separator. This is needed because for `.txt` files, the test framework treates every line as a standalone statement/command that can be executed against the server.
+- You CANNOT club functionalities of different file types. For example, you cannot execute prep-exec statements (functionality of `.txt` input file) in a `.mix` file. 
+
+## Adding tests
+The test framework consumes `.sql`, `.txt` and `.mix` files as input (discussed above) and uses them to generate the output (.out) files.
+
+1. Add the input test file in the `test/JDBC/input` directory. You can also create subdirectories and test files inside those
+2. Run the test framework to generate the output file (you may want to edit the schedule file to only the run the tests that you have added instead of running all the tests). The test will fail because it does not have a corresponding expected output file yet.
+3. Check the `test/JDBC/output` directory for the generated output file. The output file (.out) will be of the format:
+    ```
+    test/JDBC/output/<your_test_filename>.out
+    ```
+4. If the output file looks correct to you, move it to the `test/JDBC/expected` directory:
+    ```
+    mv test/JDBC/output/<your_test_filename>.out test/JDBC/expected
+    ```
+5. Re-run the test framework and ensure that the newly added test now passes
+
+## Reading the console output and diff
+If all the tests PASS, `TESTS FAILED` will be zero and you will be greeted with a `BUILD SUCCESS` message
+```
+###########################################################################
+TOTAL TESTS:	575
+TESTS PASSED:	575
+TESTS FAILED:	0
+###########################################################################
+.
+.
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+```
+
+If one or more tests FAIL, `TESTS FAILED` will not be zero and you will be greeted with a `BUILD FAILURE` message
+```
+###########################################################################
+TOTAL TESTS:	574
+TESTS PASSED:	573
+TESTS FAILED:	1
+###########################################################################
+Output diff can be found in '/home/runner/work/babelfish_extensions/babelfish_extensions/test/JDBC/Info/12-03-2045T06:07:08.009/12-03-2045T06:07:08.009.diff'
+.
+.
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD FAILURE
+[INFO] ------------------------------------------------------------------------
+```
+
+You will also be provided with the location of the `.diff` file which contains the diff between the generated output and expected output. The format of the name of the diff file is `dd-MM-yyyy'T'HH:mm:ss.SSS` where
```

Comment:
How about:

You will also be provided with the location of the `.diff` file containing the difference between the generated output and expected output. The diff file name is a date/time stamp, in the form `dd-MM-yyyy'T'HH:mm:ss.SSS` where: